eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -1980,13 +1982,24 @@ public function testBookmarksAreSwappedAfterSwapLocation()
         $beforeSwap = $bookmarkService->loadBookmarks();
 
         // Swaps the content referred to by the locations
-        $locationService->swapLocation($mediaLocation, $demoDesignLocation);
+        $locationService->swapLocation($demoDesignLocation, $contactUsLocation);
 
         $afterSwap = $bookmarkService->loadBookmarks();
         /* END: Use Case */
 
-        $this->assertEquals($beforeSwap->items[0]->id, $afterSwap->items[1]->id);
-        $this->assertEquals($beforeSwap->items[1]->id, $afterSwap->items[0]->id);
+        $expectedIdsAfter = array_map(static function (Location $item) use ($demoDesignLocationId, $contactUsLocationId) {
+            if ($item->id === $demoDesignLocationId) {
+                return $contactUsLocationId;
+            }
+
+            return $item->id;
+        }, $beforeSwap->items);
```

Comment:
Why is this mapping necessary? In particular, why is there a special case for `$demoDesignLocationId` + `$contactUsLocationId`?

Reply:
The mapping is no longer needed. In first version of PR, I did not implement `SortClause\BookmarkId`, so order of the result needed to be normalized.
I will revert the mapping.

I think the test is better when adding `SortClause\BookmarkId`. In original test we test the edge case where two bookmarked locations are swapped... In my version of the test, only one of the locations that are swapped are bookmarked.

Reply:
Done in 8483861fa87eaf7929084818623dace97d088349

ok?

Reply:
@Steveb-p @ViniTou : Could you have a second look on this please?

Reply:
@ping @Steveb-p @ViniTou

Reply:
ping @Steveb-p @ViniTou